teensy/main/main.ino

```diff
@@ -81,28 +93,41 @@ int count = 0;
 void print_all_sensors() {
   //prints a single string containing all sensor values
   //to be decoded by python script
-
+  String output = "";
   //expect precision issues to # of digits
   //python script currently expects 2 digits per datum
 
   //GPS
   Serial1.print("^latitude:");
```

Comment:
This doesn't have to be done immediately, but eventually, I'd like to standardize all of the data prefixes so that they're all consistent. They're a mix of lower, uppercase, full words, and abbreviations which is mostly my fault, but its something we should clean up in the future